### PR TITLE
Sd card and favorites

### PR DESCRIPTION
**Pull request by marioba**
- Add read-only sd card explanation
- Add favorite directories explanation

### CODE REVIEW

File: en/project-management/project-selection.rst [hunk 1]
```diff
@@ -12,14 +12,20 @@ Inside these directories, only subdirectories and files with the ".qgs" extensio
...
 External SD card
 ----------------
-The file selector only shows the application-specific directory on an external SD card, as it is the only directory for which the app has write access. The directory is the following:
+The file selector shows the root of the SD card labeled as "read-only" and the application-specific directory separated from the rest of the SD card, as it is the only directory for which the app has write access. The directory is the following:
 
 .. code-block:: none
 
     /Android/data/ch.opengis.qfield/files
 
+Technically, this is due to the fact that Android intents ACTION_OPEN_DOCUMENT and ACTION_OPEN_DOCUMENT_TREE, work and grant read-write access on the content of the returning URI, not on the real linked file. This limitation which makes sense for how Android is structured but currently it is unusable with QField as QField needs to access the real file with a real path.
```
Comment (m-kuhn) [inline en/project-management/project-selection.rst]: Can we put this in a `..note:` block, this is very developer oriented while the average user of this is probably a GIS expert.

Comment (marioba) [reply en/project-management/project-selection.rst]: yep

File: en/project-management/project-selection.rst [hunk 2]
```diff
@@ -24,6 +24,8 @@ The file selector shows the root of the SD card labeled as "read-only" and the a
 
     /Android/data/ch.opengis.qfield/files
 
+.. note::
+
 Technically, this is due to the fact that Android intents ACTION_OPEN_DOCUMENT and ACTION_OPEN_DOCUMENT_TREE, work and grant read-write access on the content of the returning URI, not on the real linked file. This limitation which makes sense for how Android is structured but currently it is unusable with QField as QField needs to access the real file with a real path.
```
Comment (m-kuhn) [inline en/project-management/project-selection.rst]: I think this needs to be indented to be under notes

Comment (marioba) [reply en/project-management/project-selection.rst]: yes, indeed.
I fixed it and all the rst2html warnings and tested locally.

**Pull request merged**